src/screens/MigrateAccounts/Banner.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import React, { useCallback } from "react";
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import type { NavigationScreenProp } from "react-navigation";
+import { compose } from "redux";
+import { withNavigation } from "react-navigation";
+import { connect } from "react-redux";
+import { createStructuredSelector } from "reselect";
+import colors from "../../colors";
+import LText from "../../components/LText";
+import IconArrowRight from "../../icons/ArrowRight";
+import LiveLogo from "../../icons/LiveLogoIcon";
+import { someAccountsNeedMigrationSelector } from "../../reducers/accounts";
+
+const mapStateToProps = createStructuredSelector({
+  someAccountsNeedMigrationSelector,
+});
+
+const Banner = ({
+  navigation,
+  someAccountsNeedMigrationSelector,
+}: {
+  someAccountsNeedMigrationSelector: boolean,
+  navigation: NavigationScreenProp<*>,
+}) => {
+  const navigateToAccountMigration = useCallback(() => {
+    navigation.navigate("MigrateAccountsOverview");
+  }, [navigation]);
+
+  if (!someAccountsNeedMigrationSelector) return null;
+
+  return (
+    <TouchableOpacity style={styles.root} onPress={navigateToAccountMigration}>
+      <View style={styles.logo}>
+        <LiveLogo size={16} color={colors.white} />
+      </View>
+      <LText semiBold style={styles.text}>
+        Ledger Live accounts update
```

Comment:
No _Actualización de cuentas de Ledger Live_ for you Spaniards :trollface:

Reply:
Tiene buen ojo, gracias